charts/cluster-api-runtime-extensions-nutanix/templates/csi/nutanix/manifests/helm-addon-installation.yaml

```diff
@@ -9,6 +9,5 @@ metadata:
 data:
   values.yaml: |-
     # The Secret containing the credentials will be created by the handler.
-    createSecret: false
-    secretName: nutanix-csi-credentials
```

Comment:
Any reason not to keep this override (renamed to `pcSecretName`)? The default Secret its looking for is `ntnx-pc-secret` and the CSI controller is getting deployed into a pretty generic `ntnx-system` namespace. The name is somewhat misleading, it looks like a Secret that can be used by any component but the format is very specific to this CSI driver.

Reply:
It seems like the only thing that gets deployed to the `ntnx-system` namespace is the CSI driver. 

I just kept the defaults to have the least amount of friction possible.

Reply:
i'm trying with the old secret name.